fastplotlib/graphics/_base.py

```diff
@@ -116,6 +116,31 @@ def link(
             callback: callable = None,
             bidirectional: bool = False
     ):
+        '''
+        Link two graphics together
+
+        Parameters
+        ----------
+        event_type: str
+            can be a pygfx event (click, etc.) or appropriate feature event associated with
```

Comment:
list of out all the events and list out a few example features. State that you can use `graphic_instance.feature_events` to get the list of valid features for a graphic.